holoviews/core/util.py

```diff
@@ -647,6 +647,30 @@ def walk_depth_first(name):
                                     (names_by_level.get(i, None)
                                      for i in itertools.count())))
 
+
+def is_cyclic(graph):
+    """Return True if the directed graph g has a cycle."""
```

Comment:
I'm guessing the representation is similar as in ``one_to_one``...even so, probably worth mentioning..